pages/keplr/first-time-flow-page.js

```diff
@@ -14,12 +14,17 @@ const phraseAccountCreated = 'Account Created!';
 const finishButton = 'button[type="button"]';
 const textAreaSelector = 'textbox';
 const submitPhraseButton = 'button[type="submit"]';
+const chainName = 'Agoric local'
+const chainNameSelector = 'text=Agoric local'
```

Comment:
we shouldnt have selectors this specific

Reply:
They're used for selecting chain and not giving any issues.

Reply:
but not every user wants the agoric chain in their setup